packages/editor/src/components/deprecated.js

```diff
@@ -117,3 +117,4 @@ export {
 	withColors,
 	withFontSizes,
 };
+export { default as ServerSideRender } from '@wordpress/server-side-render';
```

Comment:
Should we add a `deprecated` call, when using it from this location?

Reply:
I just ran into this while writing a plugin. Because `wp.components.ServerSideRender` is available with no deprecation warnings, it's easy to think that this is still the canonical place for that component.

The other thing is that the `wp-components` module hasn't listed `wp-server-side-render` as a dependency, so as a plugin author I have to know that my plugin depends on `wp-server-side-render` on top of the rest. Finally, we are in a strange situation now because those two modules are interdependent. I believe that, going forward, the only proper solution is to remove `ServerSideRender` from `components`'s deprecated components.

Of course, before that is possible, we should at least wrap `wp.components.ServerSideRender` so that it logs a deprecation warning.

Reply:
Ah, this line was about `wp-editor` and I missed that. I guess it speaks to the general confusion. :) Never mind the bit about mutual dependencies, then.

Reply:
https://github.com/WordPress/gutenberg/pull/21081